test/functional/terminal/highlight_spec.lua

```diff
@@ -206,4 +206,53 @@ describe('synIDattr()', function()
     eq('252', eval('synIDattr(hlID("Normal"), "fg")'))
     eq('79', eval('synIDattr(hlID("Keyword"), "fg")'))
   end)
+
+  it('returns "Normal" values for group using special fg/bg names', function()
```

Comment:
sure, perhaps sibling-to (not child-of) the `describe('synIDattr()',...)` block, since the purpose of these tests is not to test `synIDattr()`.